CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Sign containers using `cosign`
 * Generate and publish Software Bill of Materials (SBOMs) of Strimzi containers
 * Add support for stopping connectors according to [Strimzi Proposal #54](https://github.com/strimzi/proposals/blob/main/054-stopping-kafka-connect-connectors.md)
+* Added `strimzi.io/kraft` annotation to be applied on `Kafka` custom resource, together with the `+UseKRaft` feature gate enabled, to declare a ZooKeeper or KRaft based cluster.
+  * if `enabled` the `Kafka` resource define a KRaft cluster.
+  * if `disabled`, missing or any other value, the operator handle the `Kafka` resource as a ZooKeeper-based cluster.
```

Comment:
Maybe worth mentioning also in the `Changes, deprecations and removals` section to make sure users know that they need to change their existing clusters? I would probably:
* Phrase this as `Allow running ZooKeeper and KRaft based Apache Kafka clusters in parallel when the UseKRaft feature gate is enabled`
* List the details about the annotation that needs to be added to the KRaft YAMLs to the changes section